clerk.go

```diff
@@ -297,7 +297,7 @@ func setRequestQuery(req *http.Request, params Params) {
 	paramsQuery := params.ToQuery()
 	for k, values := range paramsQuery {
 		for _, v := range values {
-			q.Set(k, v)
+			q.Add(k, v)
```

Comment:
@georgepsarakis I think you've voiced a concern about "array" parameters in the query string. Your instinct was right. We need to add values and not overwrite them in order to support arrays.